### ISSUE TITLE
Add view for desktop events

### ISSUE THREAD

**Issue by jklukas** [mozilla/bigquery-etl#421]
@sunahsuh has mentioned that we want a view for desktop events, pulling in both main ping and event pings.

As an aside, I wanted to see what it would look like to rename fields inside the `main` view itself so that event fields have nice deanonymized names. It's quite verbose and ends up needing a lot of subqueries:

```
SELECT
  * REPLACE( (
    SELECT
      AS STRUCT payload.* REPLACE( (
        SELECT
          AS STRUCT payload.processes.* REPLACE( (
            SELECT
              AS STRUCT payload.processes.parent.* REPLACE( 
              ARRAY(SELECT
                  `moz-fx-data-shared-prod.udf.deanonymize_event`(e)
                FROM
                  UNNEST(payload.processes.parent.events) e) AS events)) AS parent)) AS processes)) AS payload)
FROM
  `moz-fx-data-shared-prod.telemetry_stable.main_v4`
WHERE
  DATE(submission_timestamp) = '2019-10-05'
  AND ARRAY_LENGTH(payload.processes.parent.events) > 0
LIMIT
  10
```

But at least it's possible to express in a view, which is fun! Maybe it's still worth codifying that in `main`.

Anyway, a really nice event view should add a parsed timestamp field that adds in the subsession start time and transforms the value into a BQ timestamp type. We can record more thoughts in this issue.

@sunahsuh - Is this something you were planning to pursue, or is it up for grabs? Do we know what name this view should have?

**Comment by sunahsuh** [comment]
I'm working on a view that's schema-compatible with the existing `events` dataset (and considering whether it should be materialized, which I'm inclined to say yes since clustering by `event_category` feels like enough of a benefit to incur the storage costs)

Re: adding event timestamps, I'm not sure we do much that takes advantage of absolute-ish time right now, but it's certainly easy enough to add in a view :) I'm not planning on it in this current iteration at the moment, though

**Comment by sunahsuh** [comment]
Just ran a quick test aggregating a couple days of enrollment/unenrollment events on a particular experiment on the view vs the existing parquet events table:
View:
![Screen Shot 2019-10-10 at 4 43 59 PM](https://user-images.githubusercontent.com/953153/66609017-92551680-eb7d-11e9-801c-dcfde458d773.png)

Events table:
![Screen Shot 2019-10-10 at 4 44 27 PM](https://user-images.githubusercontent.com/953153/66609038-a13bc900-eb7d-11e9-97a5-3efa1f148d84.png)

Seems pretty compelling re: materializing the view

**Comment by jklukas** [comment]
This was fixed in #445
